### ISSUE TITLE
USER ISSUE: Currency charge on craft stations not refunding

### ISSUE THREAD

**Issue by XsExcess** [StrangeLoopGames/EcoIssues#7286]
**Version:** 0.7.3.3 beta

**Steps to Reproduce:**
Create a work order on a work station with currency per what ever set, cancel to refund the materials

**Expected behavior:**
It should refund the currency spent

**Actual behavior:**
Refund only materials, not currency.

**Comment by Elliander** [comment]
I agree that it should refund the "Fee per item", however I don't think it should refund the "Fee per minute." - at least not all of it. If it did, then a user could exploit that by start a 16 hour build operation then cancel the last minute to get most of the items from it and no currency cost. It would need a mechanism to keep track of how many minutes were actually used - a simple math function to calculate that at the time of cancel, really - to determine how much to refund. It would also need to make sure not to refund a player who was authorized at the time, or else a new bug could allow currency duplication.

Additionally, I'd suggest that there should be a time limit on how long a player can wait to get a "Fee per item" refund, because there's a cap of 5 open projects meaning someone could hog the table, cancel, get some items for partial completion, and avoid paying any up front fees simply by ordering more than actually needed.
